shared/@hedvig-ui/Button/button.tsx

```diff
@@ -208,10 +209,29 @@ export const Button = styled(
   ({
     icon,
     children,
+    focus,
     ...props
   }: ButtonProps & ButtonHTMLAttributes<HTMLButtonElement>) => {
```

Comment:
If you extend in the props, you should remove this one!
```suggestion
  }: ButtonProps) => {
```

Reply:
Removing of it brings strange error 
![image](https://user-images.githubusercontent.com/89198006/144599164-f968f597-4c51-44a4-b69c-8ee8e0e6c39f.png)

Reply:
That's because you're extending with `React.HTMLAttributes<HTMLButtonElement>` on `ButtonProps` - use the same one you specified here, `extends ButtonHTMLAttributes<HTMLButtonElement>